arduino/scout/NewTone.h

```diff
@@ -0,0 +1,16 @@
+#ifndef NewTone_h
+#define NewTone_h
+
+#include "KeyBuffer.h"
+#include "Arduino.h"
+
+const int SPEAKER_PINS[BUFFER_MAX] = {9, 10, 11, 12};
```

Comment:
I abuse BUFFER_MAX == the number of square wave outputs all over this code. If that's a problem, I can change it -- let me know.

Reply:
I wasn't sure whether you'd want to decouple the number of available oscillator pins from the length of the key buffer. I might suggest renaming this constant to something that indicates that it's the maximum number of notes that could be playing or things that could be oscillating.